README.md

```diff
@@ -5,6 +5,8 @@ Briefly, the web needs new features, and iteration yields the best designs and i
 
 One of the root causes was that experimental features were available too widely, and thus usage grew unchecked as a result. Ideally, it should be easier to expose and iterate on new features, but reliably limit the experimental population. With a test population of developers committed to providing feedback, and limits in user base size and experiment duration, iteration can happen faster, but without the risk of burn-in.
 
+The origin trial feature is still security-reviewed, tested and launched as a production feature, just time and usage limited to allow for evolving the feature with developer feedback, minimizing the risk of it prematurely becoming a defacto standard.
+
```

Comment:
I think this is out of context here, as the readme doesn't have any detail about how origin trials work. It seems like it would be more confusing than helpful?

Reply:
It talks about the need to run limited trials to experiment and iterate on a feature but I wanted to make it clear that the features themselves are considered production-ready and it is the API surface that is being iterated on with developers (and without origin trials as a thing, would have launched as a vendor-prefixed feature).

Otherwise there's the risk of people thinking the underlying code may be risky for use on their production sites.